data_steward/bq_utils.py

```diff
@@ -739,6 +740,43 @@ def create_dataset(
     return insert_result
 
 
+def csv_line_to_sql_row_expr(row: dict, fields: list):
+    """
+    Translate a dict to a SQL row expression based on a fields spec
+
+    :param row: dict whose values are all strings
+    :param fields: bigquery fields spec with keys {name, type, mode, description}
+    :return: SQL expression for row object
+    :rtype: str
+    :example:
+    >>> fields = [{ 'name': 'int_col',  'type': 'integer', 'mode': 'required', 'description': ''},
```

Comment:
Nice thanks.

FYI: you may already be aware of this but there's a python library which can execute your code example snippets as unit tests. https://docs.python.org/3/library/doctest.html. Food for thought.